aws-crt/lib/aws-crt.rb

```diff
@@ -1,14 +1,13 @@
-module Aws
-  module Crt
-  end
-end
+# frozen_string_literal: true
 
 require_relative 'aws-crt/platforms'
 require_relative 'aws-crt/native'
 require_relative 'aws-crt/errors'
 require_relative 'aws-crt/io'
 
+# Top level Aws namespace
 module Aws
```

Comment:
I reverted to using the nested namespaces here to follow default styleguide: https://github.com/rubocop-hq/ruby-style-guide#namespace-definition